### PR TITLE
[SERVICE-320] Maximising a tabbed window from the OS-level can cause issues for the tabSet

### PR DESCRIPTION
**Pull request by bryangaleOF**
Use updateState on DesktopWindow to make window non-maximizable when added to tab group

### CODE REVIEW

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@bryangaleof). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

File: src/provider/model/DesktopWindow.ts [hunk 1]
```diff
@@ -492,14 +495,16 @@ export class DesktopWindow implements DesktopEntity {
...
         this._tabGroup = group;
 
-        // Hide tabbed windows in the task bar (except for tabstrip windows)
+        // Modify state for tabbed windows (except for tabstrip windows)
         if (this._identity.uuid !== SERVICE_IDENTITY.uuid) {
             if (group) {
-                // Hide tabbed windows in taskbar
-                return this._ready ? this.updateState({showTaskbarIcon: false}, ActionOrigin.SERVICE) : Promise.resolve();
+                // Set tabbed windows to be hidden in taskbar, and to be non-maximizable
+                const state: Partial<EntityState> = {showTaskbarIcon: false, maximizable: false};
```
Comment (tomrobertsOF) [inline src/provider/model/DesktopWindow.ts]: pedantic (sorry), but could this var be called something like `delta` instead? I think that would be a bit more clear

Comment (tomrobertsOF) [reply src/provider/model/DesktopWindow.ts]: Also, the same below.

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@bryangaleof). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

File: src/provider/model/DesktopWindow.ts (hunk 1 above)
Comment (bryangaleOF) [reply src/provider/model/DesktopWindow.ts]: Not pedantic :). This is a good change, thanks for catching

**Comment by bryangaleOF** [comment]
This has passed CI tests, one more approval and this can go in

**Pull request merged**